doc/cephfs/createfs.rst

```diff
@@ -48,7 +48,7 @@ Once the pools are created, you may enable the file system using the ``fs new``
 
 .. code:: bash
 
-    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [<fscid:int>] [--recover]
+    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [<fscid:int>] [--recover] [--yes-i-really-really-mean-it] [<set>...]
```

Comment:
Include a note that `--yes-i-really-really-mean-it` may be used for some `fs set` commands.

Reply:
added